### DISCUSSION TITLE
Kibana Machine Learning : Access Denied

### DISCUSSION THREAD

**Discussion by logz4u** [Security-Onion-Solutions/securityonion#10636] [Unsupported Versions]
Good afternoon,

When attempting to open 'Machine Learning' under Kibana -> Stack Management -> Machine Learning, I receive the message "**ACCESS DENIED** - You don't have permissions to manage Machine Learning jobs. Access to the plugins requires the Machine Learning feature to be visible in this space."

Additionally, I used the Kibana account that was generated with the SO install. I tried using standard Kibana account too, but with the same results. 

Context:
Running version 2.3.230.
I have a valid Elasticsearch Enterprise license, and NO issues uploading it into Kibana. 

Is there a special built-in account I should be using to access the Machine Learning plugin? Any direction or guidance or steps would be greatly appreciated.

**Comment by dougburks** [comment]
Have you tried enabling the feature as shown at https://docs.securityonion.net/en/2.3/kibana.html#features?

**Comment by logz4u** [comment]
Hey Doug,

Thank you for the quick response. To answer your question, the ML feature (and the other features for that matter) is enabled. Now, I went into the 'Roles' menu and noticed that the **machine_learning_admin** is reserved, and I cannot modify it. Is that role assigned to the initial Kibana user account? 

I forgot to mention that I am using an airgapped install, so would that be affecting things under the hood?
